docs/Cluster Guides/Transferring_Files.md

```diff
@@ -65,9 +65,102 @@ If your data is on an OIT Research Share, please contact Research Computing so w
+| [rclone serve restic](https://rclone.org/commands/rclone_serve_restic/)                   | Serve the remote for restic's REST API.                                 |
+| [rclone serve sftp](https://rclone.org/commands/rclone_serve_sftp/)                       | Serve the remote over SFTP.                                             |
+| [rclone serve webdav](https://rclone.org/commands/rclone_serve_webdav/)                   | Serve remote:path over WebDAV.                                          |
+| [rclone settier](https://rclone.org/commands/rclone_settier/)                             | Changes storage class/tier of objects in remote.                        |
+| [rclone sha1sum](https://rclone.org/commands/rclone_sha1sum/)                             | Produces an sha1sum file for all the objects in the path.               |
+| [rclone size](https://rclone.org/commands/rclone_size/)                                   | Prints the total size and number of objects in remote:path.             |
+| [rclone sync](https://rclone.org/commands/rclone_sync/)                                   | Make source and dest identical, modifying destination only.             |
+| [rclone test](https://rclone.org/commands/rclone_test/)                                   | Run a test command                                                      |
+| [rclone test changenotifiy](https://rclone.org/commands/rclone_test_changenotify/)        | Log any change notify requests for the remote passed in.                |
+| [rclone test histogram](https://rclone.org/commands/rclone_test_histogram/)               | Makes a histogram of file name characters.                              |
+| [rclone test info](https://rclone.org/commands/rclone_test_info/)                         | Discovers file name or other limitations for paths.                     |
+| [rclone test makefile](https://rclone.org/commands/rclone_test_makefile/)                 | Make files with random contents of the size given                       |
+| [rclone test makefiles](https://rclone.org/commands/rclone_test_makefiles/)               | Make a random file hierarchy in a directory                             |
+| [rclone test memory](https://rclone.org/commands/rclone_test_memory/)                     | Load all the objects at remote:path into memory and report memory stats.|
+| [rclone touch](https://rclone.org/commands/rclone_touch/)                                 | Create new file or change file modification time.                       |
+| [rclone tree](https://rclone.org/commands/rclone_tree/)                                   | List the contents of the remote in a tree like fashion.                 |
+| [rclone version](https://rclone.org/commands/rclone_version/)                             | Show the version number.                                                |
```

Comment:
This info is available in the docs that you linked, so I think it is better for folks to read it there. That way if they are using a different rsync version and the commands change, we don't have to update this.
```suggestion
```